nixos/modules/services/networking/newt.nix

```
@@ -6,39 +6,44 @@
 }:
 let
   cfg = config.services.newt;
+  type =
+    with lib.types;
+    attrsOf (
+      nullOr (oneOf [
+        bool
+        int
+        float
+        str
+        path
+        (listOf cliType)
```

```suggestion
        (listOf type)
```
This is why we avoid `with`.